Misc/NEWS.d/next/Windows/2019-08-06-18-09-18.bpo-25172.Akreij.rst

```diff
@@ -0,0 +1 @@
+Trying to import the crypt module on Windows will result in an ImportError with a message explaining that the module isn't supported on Windows. On other platforms, if the underlying _crypt module is not available, the ImportError will include a message explaining the problem.
```

Comment:
```suggestion
Trying to import the :mod:`crypt` module on Windows will result in an :exc:`ImportError` with a message explaining that the module isn't supported on Windows. On other platforms, if the underlying ``_crypt`` module is not available, the ImportError will include a message explaining the problem.
```

This will add cross-linking from the NEWS page to the modules.